# For timestamps
Header header

uint8 walkingState
float64 q
